misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickStepButton.jsx

```diff
@@ -95,6 +114,7 @@ PickStepButton.propTypes = {
   qtyToPick: PropTypes.number.isRequired,
   stepState: PropTypes.object,
   onHandleClick: PropTypes.func.isRequired,
```

Comment:
group this together with other functions.

Reply:
Done